src/engine/core/fs/ifile_device.h

```diff
@@ -8,7 +8,7 @@ namespace FS
 {
 
 
-class LUMIX_ENGINE_API IFile;
```

Comment:
This gave warning `type attributes ignored after type is already defined`. Besides this change will unify declarations of `IFile`.